### PR TITLE
More info for Crowdin usage

### PR DESCRIPTION
**Pull request by pierallard**
| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

### CODE REVIEW

File: contribute_to_pim/translate.rst [hunk 1]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+You want to use a non fully translated language in your application?
+--------------------------------------------------------------------
 
-How it works
-------------
+.. note::
 
-When a new translation key is added in the code of the PIM, this key is sent to Crowdin.
+    We strongly encourage you to use the default way, that is validate the application in more than 80% and wait for the next patch.
+    Indeed, an application not fully translated will contains a lot of non translated strings in many places of your application and can result of a very bad user experience.
```
Comment (ahocquard) [inline contribute_to_pim/translate.rst]:
```suggestion
    Indeed, an application not fully translated will contain a lot of non translated strings in many places of your application and can result in a very bad user experience.
```

File: contribute_to_pim/translate.rst [hunk 2]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
 
-Keys are translated by contributors, then validated by proof readers.
+By default, Akeneo PIM only displays and synchronize languages validated in more than 80%.
```
Comment (ahocquard) [inline contribute_to_pim/translate.rst]:
```suggestion
By default, Akeneo PIM only displays and synchronizes languages validated in more than 80%.
```
Also, " in more than 80%" seems weird, any English reviewer to suggest a better thing?

File: contribute_to_pim/translate.rst [hunk 3]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+If you have access to the code of your application, you can however set your user interface with another language:
 
-Each week, we download validated translations from Crowdin to update the PIM.
+Start by downloading the complete archive of all the managed translation in Crowdin `here <https://crowdin.com/backend/download/project/akeneo.zip>`_ and extract it.
```
Comment (ahocquard) [inline contribute_to_pim/translate.rst]:
```suggestion
Start by downloading the complete archive of all the managed translations in Crowdin `here <https://crowdin.com/backend/download/project/akeneo.zip>`_ and extract it.
```

File: contribute_to_pim/translate.rst [hunk 4]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
 
 .. image:: ./crowdin-logo.png
 
-How to translate
-----------------
+How to translate?
+-----------------
 
-Don't hesitate to create an account and begin to translate on: https://crowdin.com/project/akeneo
+You have to to create an account and begin to translate in your favorite language on `Crowdin`_.
```
Comment (grena) [inline contribute_to_pim/translate.rst]: You have to ~to~ create

File: contribute_to_pim/translate.rst [hunk 5]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+You need a proofreading permission to validate the strings, so your translation will be included in the PIM.
+Each language with more than 80% of valid translations are automatically translated with `Nelson`_ each week, and will be available in the next patch.
+You will be able to claim your badge "El Translator" on `Badger platform <http://badger.akeneo.com/login/>`_!
 
-If your language is not enabled, don't hesitate to ask, we'll enable it with pleasure.
+Your language is not available on Crowdin?
+------------------------------------------
 
-If you want to become a proofreader and be able to validate translations for a language, don't hesitate to ask.
+As Akeneo PIM is Open Source, we will be glad to add welcome new languages for our application.
```
Comment (grena) [inline contribute_to_pim/translate.rst]: we will be glad to ~add~ welcome

File: contribute_to_pim/translate.rst [hunk 6]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+Don't hesitate to ask to a `Crowdin`_ administrator to add your language to the available languages list of Crowdin.
+Please reminder that only languages validated in more than 80% will be synchronized in Akeneo PIM.
```
Comment (grena) [inline contribute_to_pim/translate.rst]: ~Please reminder~ We remind you that

File: contribute_to_pim/translate.rst [hunk 7]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
 
-Don't hesitate to claim your badge "El Translator" on `Badger platform <http://badger.akeneo.com/login/>`_!
+You want to use a non fully translated language in your application?
```
Comment (grena) [inline contribute_to_pim/translate.rst]: How to use a partially translated language in your application?

File: contribute_to_pim/translate.rst [hunk 8]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+--------------------------------------------------------------------
 
-How it works
-------------
+.. note::
 
-When a new translation key is added in the code of the PIM, this key is sent to Crowdin.
+    We strongly encourage you to use the default way, that is validate the application in more than 80% and wait for the next patch.
```
Comment (grena) [inline contribute_to_pim/translate.rst]: I don't understand this sentence

File: contribute_to_pim/translate.rst (hunk 1 above)
Comment (grena) [inline contribute_to_pim/translate.rst]: "Indeed, a partially translated PIM can lead to a very bad user experience."

File: contribute_to_pim/translate.rst (hunk 2 above)
Comment (grena) [inline contribute_to_pim/translate.rst]: synchronizes

File: contribute_to_pim/translate.rst (hunk 3 above)
Comment (grena) [inline contribute_to_pim/translate.rst]: translations

File: contribute_to_pim/translate.rst [hunk 9]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
 
-Keys are translated by contributors, then validated by proof readers.
+By default, Akeneo PIM only displays and synchronize languages validated in more than 80%.
+If you have access to the code of your application, you can however set your user interface with another language:
 
-Each week, we download validated translations from Crowdin to update the PIM.
+Start by downloading the complete archive of all the managed translation in Crowdin `here <https://crowdin.com/backend/download/project/akeneo.zip>`_ and extract it.
 
-We deliver the new translations in the next patch for each version.
+Then, copy the translations in the right folders. For example, if you are using 3.0 version:
```
Comment (grena) [inline contribute_to_pim/translate.rst]: in the correct folder depending on the version you use?

File: contribute_to_pim/translate.rst [hunk 10]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
 
-This workflow is handled by our open source tool Nelson: https://github.com/akeneo/nelson
+.. code-block:: bash
 
-.. image:: ./nelson.png
+    cp -r crowdin_archive/3.0/Community/* your_app/vendor/akeneo/pim-community-dev/
 
-.. _Crowdin: https://crowdin.com/project/akeneo
+If you are in Enterprise Edition, run this command too:
```
Comment (grena) [inline contribute_to_pim/translate.rst]: If you use the

File: contribute_to_pim/translate.rst (hunk 4 above)
Comment (pierallard) [inline contribute_to_pim/translate.rst]:
```suggestion
You have to create an account and begin to translate in your favorite language on `Crowdin`_.
```

File: contribute_to_pim/translate.rst [hunk 11]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+You need a proofreading permission to validate the strings, so your translation will be included in the PIM.
+Each language with more than 80% of valid translations are automatically translated with `Nelson`_ each week, and will be available in the next patch.
+You will be able to claim your badge "El Translator" on `Badger platform <http://badger.akeneo.com/login/>`_!
 
-If your language is not enabled, don't hesitate to ask, we'll enable it with pleasure.
+Your language is not available on Crowdin?
+------------------------------------------
 
-If you want to become a proofreader and be able to validate translations for a language, don't hesitate to ask.
+As Akeneo PIM is Open Source, we will be glad to add welcome new languages for our application.
```
Comment (pierallard) [inline contribute_to_pim/translate.rst]:
```suggestion
As Akeneo PIM is Open Source, we will be glad to welcome new languages for our application.
```

File: contribute_to_pim/translate.rst [hunk 12]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+You will be able to claim your badge "El Translator" on `Badger platform <http://badger.akeneo.com/login/>`_!
 
-If your language is not enabled, don't hesitate to ask, we'll enable it with pleasure.
+Your language is not available on Crowdin?
+------------------------------------------
 
-If you want to become a proofreader and be able to validate translations for a language, don't hesitate to ask.
+As Akeneo PIM is Open Source, we will be glad to welcome new languages for our application.
+Don't hesitate to ask to a `Crowdin`_ administrator to add your language to the available languages list of Crowdin.
+Please reminder that only languages validated in more than 80% will be synchronized in Akeneo PIM.
```
Comment (pierallard) [inline contribute_to_pim/translate.rst]:
```suggestion
We remind you that only languages validated in more than 80% will be synchronized in Akeneo PIM.
```

File: contribute_to_pim/translate.rst [hunk 13]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+Your language is not available on Crowdin?
+------------------------------------------
 
-If you want to become a proofreader and be able to validate translations for a language, don't hesitate to ask.
+As Akeneo PIM is Open Source, we will be glad to welcome new languages for our application.
+Don't hesitate to ask to a `Crowdin`_ administrator to add your language to the available languages list of Crowdin.
+We remind you that only languages validated in more than 80% will be synchronized in Akeneo PIM.
 
-Don't hesitate to claim your badge "El Translator" on `Badger platform <http://badger.akeneo.com/login/>`_!
+You want to use a non fully translated language in your application?
```
Comment (pierallard) [inline contribute_to_pim/translate.rst]:
```suggestion
How to use a partially translated language in your application?
```

File: contribute_to_pim/translate.rst [hunk 14]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
-Don't hesitate to claim your badge "El Translator" on `Badger platform <http://badger.akeneo.com/login/>`_!
+How to use a partially translated language in your application?
+--------------------------------------------------------------------
 
-How it works
-------------
+.. note::
 
-When a new translation key is added in the code of the PIM, this key is sent to Crowdin.
+    We strongly encourage you to use the default way, that is validate the application in more than 80% and wait for the next patch.
```
Comment (pierallard) [inline contribute_to_pim/translate.rst]:
```suggestion
    We strongly encourage you to use the default way: validate translations in more than 80% and wait for the next patch.
```

File: contribute_to_pim/translate.rst [hunk 15]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+Then, copy the translations in the right folders. For example, if you are using 3.0 version:
 
-This workflow is handled by our open source tool Nelson: https://github.com/akeneo/nelson
+.. code-block:: bash
 
-.. image:: ./nelson.png
+    cp -r crowdin_archive/3.0/Community/* your_app/vendor/akeneo/pim-community-dev/
 
-.. _Crowdin: https://crowdin.com/project/akeneo
+If you are in Enterprise Edition, run this command too:
```
Comment (pierallard) [inline contribute_to_pim/translate.rst]:
```suggestion
If you use the Enterprise Edition, run this command too:
```

File: contribute_to_pim/translate.rst [hunk 16]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+How to use a partially translated language in your application?
+--------------------------------------------------------------------
 
-How it works
-------------
+.. note::
 
-When a new translation key is added in the code of the PIM, this key is sent to Crowdin.
+    We strongly encourage you to use the default way: validate translations in more than 80% and wait for the next patch.
+    Indeed, an application not fully translated will contain a lot of non translated strings in many places of your application and can result in a very bad user experience.
```
Comment (pierallard) [inline contribute_to_pim/translate.rst]:
```suggestion
    Indeed, an application not fully translated will contain a lot of non translated strings in many places of your application and can lead to a very bad user experience.
```

File: contribute_to_pim/translate.rst [hunk 17]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+How to use a partially translated language in your application?
+--------------------------------------------------------------------
 
-How it works
-------------
+.. note::
 
-When a new translation key is added in the code of the PIM, this key is sent to Crowdin.
+    We strongly encourage you to use the default way: validate translations in more than 80% and wait for the next patch.
+    Indeed, an application not fully translated will contain a lot of non translated strings in many places of your application and can lead to a very bad user experience.
```
Comment (pierallard) [inline contribute_to_pim/translate.rst]:
```suggestion
    Indeed, a partially translated PIM will contain a lot of non translated strings in many places of your application and can lead to a very bad user experience.
```

File: contribute_to_pim/translate.rst [hunk 18]
```diff
@@ -5,31 +5,60 @@ We use `Crowdin`_ to translate the PIM.
...
+------------------------------------------
 
-If you want to become a proofreader and be able to validate translations for a language, don't hesitate to ask.
+As Akeneo PIM is Open Source, we will be glad to welcome new languages for our application.
+Don't hesitate to ask to a `Crowdin`_ administrator to add your language to the available languages list of Crowdin.
+We remind you that only languages validated in more than 80% will be synchronized in Akeneo PIM.
 
-Don't hesitate to claim your badge "El Translator" on `Badger platform <http://badger.akeneo.com/login/>`_!
+How to use a partially translated language in your application?
+--------------------------------------------------------------------
```
Comment (pierallard) [inline contribute_to_pim/translate.rst]:
```suggestion
---------------------------------------------------------------
```

**Pull request merged**